software/programming_instructions.md

```diff
@@ -77,6 +77,19 @@ Use the exact same process as for the ssd1306 library to install the europi libr
 3. Click 'Install'.
 4. You will now see several new files, including 'europi.py' alongside the 'ssd1306.py' inside the 'lib' folder.
 
+### (Optional) Installing the EuroPi Contrib library
+
+The EuroPi Contrib library will make user-contributed software available when using the [Menu](/software/contrib/menu.md) software. To install it, follow the same steps as the previous libraries on Thonny:
+
+1. Type 'micropython-europi-contrib' into the search box and click 'Search on PyPi'
+2. Click the result named 'micropython-europi-contrib'.
+
+    ![Screenshot from 2023-07-14 03-02-02](https://github.com/Allen-Synthesis/EuroPi/assets/5189714/6690e1e3-56e1-49d6-8701-6f5912d10ba1)
+   
+4. Click 'Install'.
```

Comment:
"3." was skipped in the markdown. Alternatively, markdown allows you to use "1." for each ordered list item and it will render the correct numbered order.

Reply:
Oops, my bad. Fixed it. Thanks for the tip!